deps/rabbit/src/rabbit_channel_tracking.erl

```diff
@@ -116,27 +110,14 @@ handle_cast({user_deleted, Details}) ->
     _ = timer:apply_after(?TRACKING_EXECUTION_TIMEOUT, ?MODULE,
             delete_tracked_channel_user_entry, [Username]),
     ok;
-handle_cast({node_deleted, Details}) ->
-    case rabbit_feature_flags:is_enabled(tracking_records_in_ets) of
-        true ->
-            ok;
-        false ->
-            Node = pget(node, Details),
-            rabbit_log_channel:info(
-              "Node '~ts' was removed from the cluster, deleting"
-              " its channel tracking tables...", [Node]),
-            delete_tracked_channels_table_for_node(Node),
-            delete_per_user_tracked_channels_table_for_node(Node)
-    end.
+handle_cast({node_deleted, _Details}) ->
+    ok.
```

Comment:
It looks like we can delete this `handle_cast/1` clause altogether and also delete https://github.com/rabbitmq/rabbitmq-server/blob/7606bcfd17202a3a42c4a012a41c2a6f31f57fa3/deps/rabbit/src/rabbit_channel_tracking_handler.erl#L54-L57